gulpfile.js

```diff
@@ -94,7 +99,7 @@ gulp.task('testLib_NodeModules', ['testLib'], function (cb) {
         .pipe(gulp.dest(path.join(_testRoot, 'lib/node_modules/vsts-task-lib')));
 });
 
-gulp.task('testResources', ['testLib_NodeModules', 'ps1tests']);
+gulp.task('testResources', ['testLib_NodeModules', 'ps1tests', 'copyTestData']);
```

Comment:
@ericsciple I'm hesitant to remove the existing "testResources" task. The gulp script uses these "top level" tasks to specify a dependency relationship on the individual tasks. testRsources is actually used lower down in by the "test" task.